enterprise/backend/src/metabase_enterprise/serialization/v2/extract.clj

```diff
@@ -162,28 +162,20 @@
   [[serdes.base/serdes-descendants]] is recursively called on these entities and all their descendants, until the
   complete transitive closure of all descendants is found. This produces a set of `[\"ModelName\" id]` pairs, which
   entities are then extracted the same way as [[extract-metabase]].
-
-  If the `:selected-collections` option is set, this function will emit warnings if any transitive descendants belong to
-  Collections not in the `:selected-collections` set. If `:on-error` is `\"abort\"` (the default) this will throw an
-  exception that stops the serialization process. If `:on-error` is `\"continue\"` it will keep emitting warnings, and
-  simply skip over the offending entities (recursively). Eg. if Dashboard B includes a Card A that is derived from a
+Eg. if Dashboard B includes a Card A that is derived from a
   Card C that's in an alien collection, warnings will be emitted for C, A and B, and all three will be excluded from the
   serialized output."
-  [{:keys [selected-collections targets] :as opts}]
```

Comment:
The `selected-collections` key was only ever passed by the API, which also passed an identical set of collection IDs in `targets`, so, just use `targets`.